src/reverse_route.coffee

```diff
@@ -1,11 +1,22 @@
-class ReverseRoute extends String
+class ReverseRoute
```

Comment:
Yeah this always felt weird. A Route is not a special type of string. This line always violated the Liskow substitution principle, and I'm glad to see it gone.